src/Common/tests/System/PlatformDetection.cs

```diff
@@ -456,5 +456,28 @@ public bool IsInRange(Version version)
                 return version >= Start && (Finish == null || version <= Finish);
             }
         }
+
+        public static bool IsEnvironmentCommandLineArgsSupported => s_IsEnvironmentCommandLineArgsSupported.Value;
+
+        private static readonly Lazy<bool> s_IsEnvironmentCommandLineArgsSupported = new Lazy<bool>(
+            delegate ()
+            {
+                if (!PlatformDetection.IsNetNative)
+                    return true;
+
+                try
+                {
+                    Environment.GetCommandLineArgs();
+                }
+                catch (PlatformNotSupportedException)
+                {
+                    return false;
+                }
+                catch
+                {
+                    return true; // Hmm - threw something but not PNSE. Let the tests loose on it and report what they find...
```

Comment:
"and when" - does that indicate a strong (and in >50%) possibility to support it in the future? Because if so, we might as well leave it ActiveIssue'd (and change the contents of the issue.)